forge/src/main/java/whocraft/tardis_refined/common/data/LangProviderEnglish.java

```diff
@@ -112,6 +112,10 @@ protected void addTranslations() {
 
         /*Tool Tips*/
         add(ModMessages.TOOLTIP_TARDIS_LIST_TITLE, "Key Set:");
+
+        /*Config*/
+        add(ModMessages.CONFIG_IP_COMPAT, "Immersive Portals Compatibility?");
+        add(ModMessages.CONFIG_CONTROL_NAMES, "Render control names?");
```

Comment:
Probably doesn't need the ?

Reply:
Translation is for UIs, the ? makes sense in the context of a UI

Reply:
![image](https://user-images.githubusercontent.com/34910888/211170664-a6bae8a2-3ce0-4394-bf1a-fc8a604a4182.png)